src-tests/Process/KeyValueCapabilityOptionsParserTest.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+namespace Lmc\Steward\Process;
+
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Console\Exception\RuntimeException;
+
+/**
+ * @covers Lmc\Steward\Process\KeyValueCapabilityOptionsParser
+ */
+class KeyValueCapabilityOptionsParserTest extends TestCase
+{
+    /**
+     * @dataProvider provideCapabilities
+     * @param string $capabilityOption
+     * @param string $expectedCapabilities
```

Comment:
According to provider `capabilityOption` and `expectedCapabilities` should be arrays.